### PR TITLE
[puzzmos] change to template MangaReaderCMS

### LINKED ISSUE #1610: update theme puzzmos
> **From:** 

**Version:** [](https://github.com/manga-download/hakuneko/commits/)

http://puzzmos.com/
manga site in Turkish
theme: my manga reader

<details><summary>What has been done?</summary>

</details>

### PR DESCRIPTION
**Pull request by XenonR**
Fixes #1610

### CODE REVIEW

File: src/web/mjs/connectors/Puzzmos.mjs [hunk 1]
```diff
@@ -1,85 +1,12 @@
...
-/**
- * Seems to be customized FlatManga CMS
- * Very similar to EpikManga
- */
-export default class Puzzmos extends Connector {
-
-    /**
-     *
-     */
+export default class Biamam extends MangaReaderCMS {
```
Comment (27136773) [inline src/web/mjs/connectors/Puzzmos.mjs]: Change class name, there is already another connector with that name

**Review by 27136773 — CHANGES_REQUESTED** [review]
Need to resolve class name collision (probably copy/pasted)

File: src/web/mjs/connectors/Puzzmos.mjs (hunk 1 above)
Comment (XenonR) [reply src/web/mjs/connectors/Puzzmos.mjs]: yes, copy/paste.

**Review by 27136773 — APPROVED** [review]
Looks good

**Pull request merged**